fix: fix and update the tutorials for `0.17` (#744)
cosmos/sdk-tutorials: liquidity-module/index.md

gaiad keys add username --recover
```

> It will ask you for the passphrase, that you see in the terminal window of running `starport chain serve` on your `myblockchaind`

Reviewer: ```suggestion
The command prompts you for the passphrase. You can find the passphrase in the terminal window that is running `starport chain serve` on your `myblockchaind`
```
